packages/search/src/users/errors.ts

```diff
@@ -0,0 +1,15 @@
+import { getProp } from "@esri/hub-common";
+
+export class GraphQLError extends Error {
+  constructor(private errors: Error[]) {
+    /* istanbul ignore next */
+    super();
+    Object.setPrototypeOf(this, GraphQLError.prototype);
```

Comment:
TypeScript cannoto extend certain built-ins, see [this breaking change](https://github.com/Microsoft/TypeScript-wiki/blob/master/Breaking-Changes.md#extending-built-ins-like-error-array-and-map-may-no-longer-work). Setting the prototype lets us override `Error` with a custom-formatted error message.